config/config.go

```diff
@@ -688,23 +689,6 @@ func NewConfig() (c *Config) {
 			Istio: IstioConfig{
 				ComponentStatuses: ComponentStatuses{
 					Enabled: true,
-					Components: []ComponentStatus{
-						{
-							AppLabel: "istio-egressgateway",
-							IsCore:   false,
-							IsProxy:  true,
-						},
-						{
-							AppLabel: "istio-ingressgateway",
-							IsCore:   true,
-							IsProxy:  true,
-						},
-						{
-							AppLabel: "istiod",
-							IsCore:   true,
-							IsProxy:  false,
-						},
-					},
```

Comment:
> Functionality of external_services.istio.component_status.components is left as deprecated for now, until will be removed, or will left as empty for possible custom components such as eastwestgateway.

Do we really want to leave this unused Config?  If we think it's going to be useful in the near future, then maybe, but otherwise I'm not sure it's worth keeping.  If nothing else I think a comment may be appropriate in the type definition.

Reply:
The reason it is left is, that there can be components such as "istio-eastwestgateway", or any other custom component users might want to be checked. So far we can mark it as deprecated.